grid/Grid-Recon.md

```diff
@@ -0,0 +1,39 @@
+
+## Grid Reconnaisance
+This is mainly to discover the different services, ports, technologies the infrastructure uses.
+This helps understand the underlying technologies, ports and services so we can search for the versions which are vulnerable, try to abuse `wrongly configured settings, try to gain control, escalade, bypass, fuzz` etc.
+Tools used :
+- `kali OS` with pre-installed tools. But you may need to manually install other custom tools like `ffuf` etc.
+- Nmap
+- Nikto
+- Dirb
+#### 1. Nmap : Used for network scanning, port discovery, services using NSE scripts and many more.
+- `5 ports` opened, `5000`, `5001`, `5002`, `5003`, `7000`.
+-  Ports `5000-5003` and `7000` uses `Ajenti http control panel` which is an open source project maintained by : http://ajenti.org/
```

Comment:
What are the consequences of this? See "summary" section, below

Reply:
Updated